### DISCUSSION TITLE
Do you receive data normally while charging?

### DISCUSSION THREAD

**Discussion by 5kat3R** [flobz/psa_car_controller#746] [General]
Hi, 

For last few days the data from the vehicle doesn't get updated while it's charging but works normally when I drive the car.
And because the data doesn't get updated, the set percentage limit 80% doesn't work and the battery is charged to 100%

Has anyone else experienced the same problem? I recently terminated my Eevee app account and now I'm wondering could that be the reason for this issue?

**Comment by Drastef31** [comment]
Hi,

Same thing for me. It's been a while that I saw that the car does not update the data when not driven (not only while charging), but until now, I was able to wake up the car with the suitable command in PSA_CAR_CONTROLLER API (I have a script in my home automation system that send the wake up command when the car data are older than 30 minutes). But since few weeks, it seems that the wake up command does not work anymore.

**Reply by 5kat3R** [reply]
In some topic someone mentioned that even the addon otherwise works normally, the solution would be to re-login.
So yesterday I uninstalled and reinstalled the addon and made a fresh login by following the new procedure.
Most probably I will need to charge my car in coming days, so I will let you now did it solve the problem or not.

**Comment by 5kat3R** [comment]
I can confirm that reinstalling PSACC addon and re-login solved the problem.
![Screenshot_20240304-064508_Home_Assistant](https://github.com/flobz/psa_car_controller/assets/52790629/02e3c557-684e-4f88-a130-fcd482f7683b)

**Comment by Drastef31** [comment]
Indeed, same think for me. PSA car controller has fully crashed (I do not know why), and after reinstalling from scratch, it is possible to wake up the car and to get fresh data.
Personnaly, I'm fed up with this instable system and I will look for something else.
thank you for your feedback
